CHANGELOG.md

```diff
@@ -81,737 +137,3 @@
 - Removed the "-beta" suffix.
 - Added various new services.
 - Added ./version package for centralized SDK versioning.
-
-## `v12.5.0-beta`
```

Comment:
Why are we getting rid of all this?

Reply:
Because it clutters up the CHANGELOG with pre-release stuffs.  If people really want this info they can get it from history.

Reply:
okok